node/src/main.js

```diff
@@ -94,26 +95,27 @@ const getCommandOptions = (command, argv) => {
   return commandsOptions[command];
 };
 
-const parseEnvironmentVariables = (environmentVariables, sensitive) => {
+const parseVariables = (variables, sensitive, type) => {
   const result = [];
 
-  if (environmentVariables && environmentVariables.length > 0) {
-    logger.info(`Getting ${sensitive ? 'Sensitive ' : ''}Environment Variables from options:`, environmentVariables);
+  if (variables && variables.length > 0) {
+    logger.info(`Getting ${sensitive ? 'Sensitive ' : ''} Variables from options:`, variables);
```

Comment:
Maybe add a conditional `Environment Variables` vs `Terraform Variables` in this log?
Without this, the user would get the same `Getting Variables from options` log twice

Reply:
done.